### PR TITLE
feat(workflow_test*.groovy): add WORKFLOW_CLI_SHA param

### PR DESCRIPTION
**Pull request by vdice**
So a specific version can be used; also fixed funky indentation

cc @joshua-anderson

### CODE REVIEW

**Comment by vdice** [comment]
~~CI will fail until https://github.com/deis/jenkins-jobs/pull/116 is merged~~

**Comment by Joshua-Anderson** [comment]
LGTM, though I'm not familiar with the Jenkins settings parts.

File: common.groovy [hunk 1]
```diff
@@ -27,6 +27,7 @@ set -eo pipefail
 
 export WORKFLOW_CHART="workflow-${RELEASE}"
 export WORKFLOW_E2E_CHART="workflow-${RELEASE}-e2e"
+export CLI_VERSION="${WORKFLOW_CLI_SHA:0:7}"
```
Comment (Joshua-Anderson) [inline common.groovy:30]: You actually only want the first `6` characters of the SHA, which is the form the binary uses to upload: `https://storage.googleapis.com/workflow-cli/e233d34/deis-e233d34-linux-amd64`

Comment (Joshua-Anderson) [inline common.groovy:30]: And I can't count at all... don't mind me 😄

**Pull request merged**